discovery/gossiper.go

```diff
@@ -1622,7 +1622,7 @@ func (d *AuthenticatedGossiper) processNetworkAnnouncement(
 		// If the advertised inclusionary block is beyond our knowledge
 		// of the chain tip, then we'll ignore for it now.
 		d.Lock()
-		if nMsg.isRemote && isPremature(msg.ShortChannelID, 0) {
+		if nMsg.isRemote && !msg.ShortChannelID.IsFake() && isPremature(msg.ShortChannelID, 0) {
```

Comment:
Why would we get a chan ann for a zero conf channel?

Reply:
It seems that these gossiper announcements are triggered from the funding manager `addToRouterGraph` method.
It is not the announcement for six confirmations.